static/app/types/workflowEngine/automations.tsx

```diff
@@ -17,3 +19,11 @@ export interface Automation extends Readonly<NewAutomation> {
   readonly id: string;
   readonly lastTriggered: string;
 }
+
+export interface AutomationFireHistory {
+  count: number;
+  eventId: string;
+  group: Group;
+  lastTriggered: Date;
```

Comment:
json can't be a date
```suggestion
  lastTriggered: string;
```